14-minimal/README.md

```diff
@@ -0,0 +1,315 @@
+NodeJS 14 minimal container image
+=========================
+
+This container image includes Node.JS 14 on top of a minimal base image for your Node.JS 14 applications. This image is best used
+with the full-sized s2i-enabled Node.JS 14 image to build the application.
+Users can choose between RHEL, CentOS and Fedora based images.
+The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
+the CentOS images are available on [Quay.io](https://quay.io/organization/centos7),
+and the Fedora images are available in [Fedora Registry](https://registry.fedoraproject.org/).
+The resulting image can be run using [podman](https://github.com/containers/libpod).
+
+Note: while the examples in this README are calling `podman`, you can replace any such calls by `docker` with the same arguments
+
+Description
+-----------
+
+Node.js 14 available as a minimal container is a base platform for
+running various Node.js 14 applications and frameworks.
+Node.js is a platform built on Chrome's JavaScript runtime for easily building
+fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model
+that makes it lightweight and efficient, perfect for data-intensive real-time applications
+that run across distributed devices.
+
+Usage in OpenShift
+------------------
+In this example, we will assume that you are using the `ubi8/nodejs-14` image, available via `nodejs:14-ubi8` imagestream tag in Openshift
+to build the application, as well as the `ubi8/nodejs-14-minimal` image, available via `nodejs:14-ubi8-minimal` image stream
```

Comment:
I wanted to say here, that we expect you to be using these two images, hence the `as well as`. Not sure if the change to `while` would make sense here. Can you explain why it would be better?